pkg/gce-pd-csi-driver/controller.go

```diff
@@ -1200,6 +1200,12 @@ func (gceCS *GCEControllerServer) CreateSnapshot(ctx context.Context, req *csi.C
 
 func (gceCS *GCEControllerServer) createPDSnapshot(ctx context.Context, project string, volKey *meta.Key, snapshotName string, snapshotParams common.SnapshotParameters) (*csi.Snapshot, error) {
 	volumeID, err := common.KeyToVolumeID(volKey, project)
+
+	volumeIsMultiZone := isMultiZoneVolKey(volKey)
```

Comment:
Please move this after the volumeKey validation to line#1212. We would like to ensure volKey is valid before checking the multi-zone support.